keadm/cmd/keadm/app/cmd/helm/cloudcore.go

```diff
@@ -124,7 +124,7 @@ func (c *CloudCoreHelmTool) Install(opts *types.InitOptions) error {
 		return fmt.Errorf("failed to verify k8s component installed, err: %v", err)
 	}
 
-	fmt.Println("Kubernetes version verification passed, KubeEdge installation will start...")
+	fmt.Printf("Kubernetes version verification passed, KubeEdge(version: %s) installation will start...\n", opts.KubeEdgeVersion)
```

Comment:
Will it be better if you don’t add brackets? For example, `KubeEdge v1.18.0 installation will start...`

Reply:
Thanks, 
Update by review.

![image](https://github.com/user-attachments/assets/073c07af-eec7-44de-bac1-c8d0ec4fc33f)

Reply:
Is it possible that version here is empty or other wrong msg? Should be not.

Reply:
Or users can't understand what's msg meaning after KubeEdge

Reply:
@fisherxu  Combining code and experiments, version information will always emerge.

In the most extreme cases:

Incorrect version input in the command
Incorrect version information of the Keadm client
Incorrect remote version information
The version information will appear as the default version information hardcoded in the code.

Reply:
Sounds good, thanks :)